src/bonus/bonusPriceTag.tsx

```diff
@@ -0,0 +1,33 @@
+import {StarFill} from '@atb/assets/svg/mono-icons/bonus';
+import {ThemeText} from '@atb/components/text';
+import {ThemeIcon} from '@atb/components/theme-icon';
+import {StyleSheet} from '@atb/theme';
+import React, {ComponentProps} from 'react';
+import {View} from 'react-native';
+
+type Props = ComponentProps<typeof View> & {
+  price: number;
+};
+export const BonusPriceTag = ({price, ...props}: Props) => {
+  const styles = useStyles();
+
+  return (
+    <View {...props} style={[styles.container, props.style]}>
+      <ThemeText>{price}</ThemeText>
+      <ThemeIcon svg={StarFill} size="small" />
```

Comment:
Just remembered about screenreader, does the screenreader work with this? will it say "image, star"?
Should we add some accessibility labels to help users with screenreader to use this bonus point feature?